### DISCUSSION TITLE
Will this LLC work instead of Red LLC?

### DISCUSSION THREAD

**Discussion by Chris-Pr** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#400] [Q&A]
https://core-electronics.com.au/8-channel-bi-directional-logic-level-converter-txb0108.html

No red LLC available in australia so looking for alternatives, please assist, thanks!

**Comment by visualapproach** [comment]
You can run without LLC.

**Comment by visualapproach** [comment]
And NO that LLC will not work
